hack/make-rules/test.sh

```diff
@@ -263,6 +265,7 @@ runTests() {
           ${testargs[@]:+${testargs[@]}} \
         | tee ${junit_filename_prefix:+\"${junit_filename_prefix}-\$_pkg_out.stdout\"} \
         | grep \"${go_test_grep_pattern}\"" \
+          {} \
```

Comment:
the indentation here is misleading, any chance you can pull this back to make it clearer it's an arg to the wrapping `bash -c` instead of the final `grep`?